transactions/src/main/java/org/jboss/as/txn/service/ArjunaRecoveryManagerService.java

```diff
@@ -26,20 +26,25 @@
 import java.util.Collections;
 import java.util.List;
 
+import com.arjuna.ats.arjuna.recovery.RecoveryManager;
 import com.arjuna.ats.internal.jta.recovery.jts.JCAServerTransactionRecoveryModule;
 import org.jboss.as.network.ManagedBinding;
 import org.jboss.as.network.SocketBinding;
 import org.jboss.as.network.SocketBindingManager;
 import org.jboss.as.server.suspend.SuspendController;
 import org.jboss.as.txn.logging.TransactionLogger;
 import org.jboss.as.txn.suspend.RecoverySuspendController;
+import org.jboss.jbossts.xts.initialisation.RecoveryInitialisation;
+import org.jboss.jbossts.xts.recovery.participant.ba.XTSBARecoveryManager;
```

Comment:
So long as there is no dependency between tx and XTS then it sounds fine - that was the perspective I was looking from
Thanks!